tools/cmake/cores/Cortex-M33-NS.cmake

```diff
@@ -6,7 +6,6 @@ if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
     list(APPEND common_options
         "-mthumb"
         "-mcpu=cortex-m33+nodsp"
```

Comment:
Will this mcpu setting disable the FPU? The intention previously appeared to be "Please disable FPU".

Reply:
We can add `+nofp`.

Reply:
However, since the old tools did not have it, I'd rather not add it to not cause side effects.

Reply:
We touched this recently: https://github.com/ARMmbed/mbed-os/pull/14196#discussion_r565233423

fpu none comes from armclang6, not valid for Gcc Arm. we should add `+nofp` for both (should be valid as I checked).

Reply:
We shouldn't have FPU enabled on ARM, but disabled on GCC. The CPU features should be consistent between the two.

Reply:
Happy to add it if we understand that we are setting it differently than the old tools did.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/47ec6374f9a411711899e4892ddfa12f7c5f9a2c..cd6cb71c29409d0100810ea2649916dd2c102188).